lib/elixir/pages/mix-and-otp/dynamic-supervisor.md

```diff
@@ -5,161 +5,192 @@
 
 # Supervising dynamic children
 
-We have now successfully defined our supervisor which is automatically started (and stopped) as part of our application life cycle.
+We have successfully how our supervisor tree is automatically started (and stopped) as part of our the application life cycle. We can also name our buckets via the `:name` option. And we also learned that, in practice, we should always start new processes inside supervisors. Let's put these teachings in practice by ensuring our buckets are named and supervised.
 
-Remember, however, that our `KV.Registry` is both linking (via `start_link`) and monitoring (via `monitor`) bucket processes in the `handle_cast/2` callback:
+## Child specs
+
+Supervisors know how to start processes because they are given "child specifications". In our `lib/kv.ex` file, we defined a list of childreni with a single child spec:
```

Comment:
```suggestion
Supervisors know how to start processes because they are given "child specifications". In our `lib/kv.ex` file, we defined a list of children with a single child spec:
```